2-library/postges-library/src/main/resources/application-postgres-library.properties

```diff
@@ -0,0 +1,10 @@
+spring.r2dbc.url=
+spring.r2dbc.username=
+spring.r2dbc.password=
```

Comment:
create `1-documentation/...`, e.g. How to use postgres reactive library where described use of this properties
```suggestion
```